scripts/install2

```diff
@@ -3,134 +3,99 @@
 true
 set -e
 
+# Enable password-less sudo on Armbian
+sudo sed -i 's/ALL=(ALL:ALL) ALL/ALL=(ALL:ALL) NOPASSWD:ALL/' /etc/sudoers
+
 TAG_CJDNS=d2e55d58548d83940482fe1bbbe1fd36f7f1b4ef
 
 # Common functions
 source functions.sh
 
+# Normalize OS environment
+# True required to avoid odd armbian bug.  Update does work but reports missing file.
+sudo apt-get update -y || true
+
+# Dependencies
+sudo apt-get install -y gcc make curl wget
+sudo apt-get install -y net-tools || true
+
 # Get board information and set flags accordingly
-BOARD_FAMILY="Unknown"
-BOARD_NAME="Generic"
-BOARD_REVISION="?"
 CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 ./do"
 
-##TODO## Possibly use /sys/firmware/devicetree/base/model
-# Use tr to remove null byte generating warning
-BOARD_MODEL=$(tr -d '\0' < /sys/firmware/devicetree/base/model)
-BOARD_HARDWARE=$(grep Hardware /proc/cpuinfo | awk '{print $3}' | head -n 1)
+detectBoard;
 
 
+# Set some defaults
+if [[ "$ARCH" == "armv7l" ]]; then
+    #TODO# -O2 workaround. Needs to be resolved.
+    CJDNS_BUILD_CMD="sudo Seccomp_NO=1 CFLAGS=\"-O2 -s -static -Wall -march=armv7-a -mtune=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -fomit-frame-pointer -marm\" ./do"
```

Comment:
Maybe single quotes would make this cleaner?

Reply:
Maybe. But its a copy /paste from Prototype version1.

The whole cjdns build needs to be assessed and cleaned up Noted in #165